tests/conftest.py

```diff
@@ -1,17 +1,58 @@
-
 from pathlib import Path
 from tempfile import TemporaryDirectory
 
 import pytest
 
 from extra_data import RunDirectory
 from extra_data.tests.mockdata import write_file
-from extra_data.tests.mockdata.xgm import XGM
 from extra_data.tests.mockdata.motor import Motor
+from extra_data.tests.mockdata.base import DeviceBase
 
 from .mockdata.timeserver import Timeserver, PulsePatternDecoder
 
 
+# This is a more accurate representation of an XGM than the XGM class from the
+# extra_data tests (it has some incorrect property names).
```

Comment:
not for this PR, but does anything prevent us from fixin the extra_data mock class?